using-mobile-extensions/adobe-analytics-edge/product-variable.md

```diff
@@ -84,67 +84,9 @@ contextData["m.purchase"] = "1"
 
 // send the tracking call - use either a trackAction or trackState call.
 // trackAction example:
-ACPCore.trackAction("purchase", data: contextData)
+MobileCore.track(action: "purchase", data: contextData)
 // trackState example:
-ACPCore.trackState("Order Confirmation", data: contextData)
-```
-{% endtab %}
-
-{% tab title="React Native" %}
-#### JavaScript
-
-#### Syntax
-
-```jsx
-contextData["&&products"] = "Category;Product;Quantity;Price[,Category;Product;Quantity;Price]";
-```
-
-#### Example
-
-```jsx
-//create a context data dictionary
-var contextData = {};
-
-// add products, a purchase id, a purchase context data key, and any other data you want to collect.
-// Note the special syntax for products
-contextData["&&products"] = ";Running Shoes;1;69.95,;Running Socks;10;29.99";
-contextData["m.purchaseid"] = "1234567890";
-contextData["m.purchase"] = "1";
-
-// send the tracking call - use either a trackAction or TrackState call.
-// trackAction example:
-ACPCore.trackAction("purchase", contextData);
-// trackState example:
-ACPCore.trackState("Order Confirmation", contextData);
-```
-{% endtab %}
-
-{% tab title="Flutter" %}
-#### Dart
-
-#### Syntax
-
-```dart
-contextData["&&products"] = "Category;Product;Quantity;Price[,Category;Product;Quantity;Price]";
-```
-
-#### Example
-
-```dart
-//create a context data dictionary
-var contextData = {};
-
-// add products, a purchase id, a purchase context data key, and any other data you want to collect.
-// Note the special syntax for products
-contextData["&&products"] = ";Running Shoes;1;69.95,;Running Socks;10;29.99";
-contextData["m.purchaseid"] = "1234567890";
-contextData["m.purchase"] = "1";
-
-// send the tracking call - use either a trackAction or TrackState call.
-// trackAction example:
-FlutterACPCore.trackAction("purchase", data: contextData);
-// trackState example:
-FlutterACPCore.trackState("Order Confirmation", data: contextData);
+MobileCore.track(state: "Order Confirmation", data: contextData)
 ```
 {% endtab %}
 {% endtabs %}
```

Comment:
We should probably remove the sample network request below and just add the description in text👇